package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "squatter",
-    "version": "0.2.0",
+    "version": "0.2.1",
```

Comment:
Could you please either undo this change or give me write access to your fork so I can do it for you? (I'll bump the version during the release process.)